openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/FeedRepository.java

```diff
@@ -836,6 +838,16 @@ private void validateAnnouncement(Thread thread) {
     }
   }
 
+  private void validateTaskByNonBotUser(Thread thread) {
```

Comment:
@sonika-shah lets make this check in populateAssignees method

Reply:
sure @harshach but we can also place it in validateAssignees method, because it contains with the validation logic and throwing exception

Reply:
@sonika-shah sounds good. Ad it in validateAssignees itself

Reply:
changes done - added logic in validateAssignees method